lib/paperclip/storage/ftp/server.rb

```diff
@@ -58,12 +58,35 @@ def get_file(remote_file_path, local_file_path)
           connection.getbinaryfile(remote_file_path, local_file_path)
         end
 
-        def put_file(local_file_path, remote_file_path)
+        def put_file(local_file_path, remote_file_path, check_existence = true)
```

Comment:
Thanks for your feedback, the second option sounds legit, I'll refactor the code accordingly.